generate_ghidra_pyi.py

```diff
@@ -50,6 +50,45 @@ def main():
     ghidra_package = type_extractor.Package.from_package(ghidra)
     type_formatter.create_type_hints(pyi_root, ghidra_package)
 
+    setup_code = """
+from setuptools import setup
+import os
+
+packages = ['ghidra-stubs']
+
+package_data = {'': ['*']}
+
+
+
+def find_stub_files():
+    result = []
+    for root, dirs, files in os.walk('ghidra-stubs'):
+        for file in files:
+            if file.endswith('.pyi'):
+                if os.path.sep in root:
+                    sub_root = root.split(os.path.sep, 1)[-1]
+                    file = os.path.join(sub_root, file)
+                result.append(file)
+    return result
+
+setup_kwargs = {
+'name': 'ghidra-stubs',
+    'version': '%s',
+    'author': 'Tamir Bahar',
+    'packages': packages,
+    'package_data': {'ghidra-stubs': find_stub_files()},
+}
+
+
+setup(**setup_kwargs)
```

Comment:
Is there a reason for constructing the arguments separately from the call?
If not, please just call the function normally.